ixdiagnose/plugin.py

```diff
@@ -28,6 +27,7 @@ def generate_plugins_debug(percentage: int = 0, total_percentage: int = 100) ->
 
         plugins_report[plugin_name] = report
         percentage += plugin_percentage
+        send_event(int(percentage + 0.5), f'Gathered debug information for {plugin_name!r} plugin')
```

Comment:
We changed this to log what plug-in is being captured so it's easier to troubleshoot as well as be more intuitive for the end-user to know what's going on.